src/test/java/htsjdk/tribble/readers/AsciiLineReaderTest.java

```diff
@@ -49,4 +41,20 @@ public void testReadLines() throws Exception {
         assertEquals(expectedNumber, actualLines);
 
     }
+
+    @Test public void voidTestLineEndingLength() throws Exception {
+        final String input = "Hello\nThis\rIs A Silly Test\r\nSo There";
+        final InputStream is = new ByteArrayInputStream(input.getBytes());
```

Comment:
What should happen if there's a stray carriage return with no linefeed? Could you add that to the test?